docs/lsp.md

```diff
@@ -207,12 +210,24 @@ FsLSP-->>-Client: DocumentDiagnosticReport
 
 This will hopefully be handled by the LSP library. Although we might need to be able to keep track of requests in progress for the purpose of cancellation.
 
+
+#### Configuration updates
+
+We should be able to apply configuration changes that require re-creating the FCS checker. We should be able to keep the caches since the results shouldn't depend on checker settings (rather only the input Project Snapshots).
+
+
 #### Extensibility
 
 The FsLSP library should be extensible by adding custom endpoints or overriding existing ones.
 
 It should also allow to hook into background processes or events to customize behavior.
 
+The code for extending the behavior should look the same as the code for implementing the core functionality. That way it's easy to test out and then potentially contribute it directly to the library.
+
+#### Release process
+
+We should use a similar release process as we have now for FCS. Potentially also provide a pre-release package with the latest version from `main`.
```

Comment:
I'm quite sceptical about this approach. Perfecting the LSP transition for every F# project is an immense challenge. It's improbable that the team will perfect it on the first try, and waiting for the subsequent SDK for corrections won't motivate anyone to participate in this development. A nightly feed is essential here.

I earnestly urge the team to shape the LSP epic in a way that enables community members to actively participate and rectify issues specific to their codebases. While it's fine for the team to lead and make key decisions, they should genuinely strive to incorporate early contributors from the community into their process. Motivate individuals to embark on this journey with you. Simplify the testing and debugging processes for personal projects. This approach could address many unique issues through collective effort.

Regular updates, seamlessly integrated by vscode as an LSP client, could significantly accelerate progress. With ample documentation, some developers will find solutions for their specific needs.